### DISCUSSION TITLE
How to add „bad“ words

### DISCUSSION THREAD

**Discussion by enko** [rspamd/rspamd#3690] [Q&A] [answered]
Hi Folks,

I'm looking for a way to give certain words a specific score. Is there an easy way to do this?

I think I could write [my own regex rule](https://rspamd.com/doc/tutorials/writing_rules.html), but maybe there is a more easy solution?

Thanks

Tim

**Comment by Eloque** [comment, marked as answer]
I've asked just about the same question here; https://github.com/Mailu/Mailu/issues/1566#issuecomment-821826939
Repeating here, since it might be relevant;

I want to flag mail containing certain words, if the mail contains that word, add 10 to the score of the mail. But I can't get it to work. 

Made a multimap.conf

```
#override.d/multimap.conf
CONTENT_BLACKLISTED {
  type = "content";
  filter = "full";
  map = "/etc/rspamd/override.d/badwords.map";
  score = 10;
  description = "Mail contains badwords";
}
```

and a badwords.map

```
# This file is live reloaded
trialwordforspam
```

I restart;
```
docker-compose up -d --scale antispam=0
docker-compose up -d --scale antispam=1
```

And check if the the config is there;

![image](https://user-images.githubusercontent.com/4184679/115115506-1c611500-9f95-11eb-9265-d0231bbbbda7.png)

I then send a mail, containing "trialwordforspam" and the filter goes straight through: 

![image](https://user-images.githubusercontent.com/4184679/115115554-71049000-9f95-11eb-880c-c670bb81ea6e.png)

As if the rule isn't triggered at all. What am I missing here?

**Reply by adjustGirl** [reply]
that work for me (without docker)

SUBJECT_BLACKLISTED {
 type = "header";
 header = "Subject";
 map = "${LOCAL_CONFDIR}/local.d/blacklist_subject.map";
 regexp = true;
 action = "reject"; # Prefilter mode
 description = "Blacklist map for SUBJECT_BLACKLISTED";
}

and the content is (German):

blacklist_subject.map

/.*Löwen.*System macht Deutsche Bürger reich!/
/.*Viagra.*bla/

**Reply by enko** [reply]
Thanks you two for the answer! Thats exactly what I needed. And yeah, this „Höhle der Löwen“ crap hits the right spot.

**Reply by Eloque** [reply]
Doesn't work for me.
I've used that exact config, but nothing triggers it. I am stumped.

**Reply by adjustGirl** [reply]
no #override.d/multimap.conf
but #local.d/multimap.conf

work fine for me. Often tested
